build.gradle.kts

```diff
@@ -32,14 +33,14 @@ allprojects {
     }
 
     apply(from = "$gradleScriptDir/bintray.gradle")
+    apply(plugin = "com.github.dcendents.android-maven")
 }
 
 configure(subprojects
         .filter { !it.name.contains("android") }
         .filter { it.parent?.name != "samples" }
 ) {
     apply(plugin = "org.jetbrains.kotlin.jvm")
-    apply(from = "$gradleScriptDir/maven-publish.gradle")
```

Comment:
Why was this removed? It nicely configures the pom of pure kotlin libraries. I would say that it should be kept and 
`apply(plugin = "com.github.dcendents.android-maven")` should only be applied to android module. It seems weird that we are using the `android-maven` plugin for kotlin like modules

Reply:
@kamildziadek, thanks a lot, good catch.